core/lib/resources/values_manager.dart

```diff
@@ -0,0 +1,48 @@
+class ApplicationPadding {
+  static const double p4 = 4;
```

Comment:
naming . it should be like this 
`
  static const double PADDING_2 = 2.0;
  static const double PADDING_4 = 4.0;
  static const double PADDING_6 = 6.0;
  static const double PADDING_8 = 8.0;`